pkg/crypto/aes_gcm.go

```diff
@@ -0,0 +1,73 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// EncryptAESGCM performs AES-GCM encryption on the given text and returns the encrypted
+// bytes as a base64 standard encoded string. The encryption key should be a base64 standard encoded string.
+func EncryptAESGCM(keyString string, stringToEncrypt string) (string, error) {
```

Comment:
Might be good to leave this funcs as Encrypt and Decryptand build an interface for it, and have this as the AES GCM implementation of that interface. Later if we choose another encryption standard, we implement another class for it.